README.md

```diff
@@ -7,13 +7,24 @@ The core abstraction in Aqueduct is a [Workflow](https://docs.aqueducthq.com/wor
 The input Artifact(s) for a Workflow is typically loaded from a database, and the output Artifact(s) are typically persisted back to a database. 
 Each Workflow can either be run on a fixed schedule or triggered on-demand. 
 
-To get started with Aqueduct, simply run the following commands from your terminal:
-
-```bash
-pip3 install aqueduct-ml
-aqueduct server &
-aqueduct ui &
-```
+To get started with Aqueduct:
+1. Ensure that you meet the [basic requirements](https://docs.aqueducthq.com/installation-and-deployment/installing-aqueduct).
+2. Install the aqueduct server and UI by running: 
+    ```bash
+    pip3 install aqueduct-ml
+    ```
+3. Launch the server by running 
```

Comment:
Nit: Can we be consistent about using colons before the code blocks?

Reply:
Yes. good catch.